pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -144,6 +148,11 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
   const [enableAutocomplete, setEnableAutocomplete] = useLocalStorage('enable-autocomplete', true);
   const [enableHighlighting, setEnableHighlighting] = useLocalStorage('enable-syntax-highlighting', true);
   const [enableLinter, setEnableLinter] = useLocalStorage('enable-linter', true);
+  const [engine, setEngine] = useLocalStorage<string>('engine', 'prometheus');
+  const engineOptions = [
```

Comment:
I think we can skip this completely and just render it, instead of having a dedicated component, as mentioned above. 🙂